ale/base/data_naif.py

```diff
@@ -80,6 +88,21 @@ def focal2pixel_samples(self):
     def _focal_length(self):
         return float(spice.gdpool('INS{}_FOCAL_LENGTH'.format(self.ikid), 0, 1)[0])
 
+    @property
+    def pixel_size(self):
+        return spice.gdpool('INS{}_PIXEL_SIZE'.format(self.ikid), 0, 1)[0] * 0.001
+
+    @property
+    def _radii(self):
```

Comment:
Why is this private?

Reply:
Because I am still not fully aware when things need to be private or not in this project and was simply following the structure from _semimajor and _semiminor below... I can get this changed. Also, working with @paarongiroux on the cassini driver, it is looking like this method is returning what the to_usgscsm method is expecting to return from it's call to "target_body_radii". Should I rename this to "target_body_radii" here?